content/yaml-quick-start/building-a-unity-app.md

```diff
@@ -491,12 +497,25 @@ workflows:
           - name: Set build number and export Unity
             script: |
               export NEW_BUILD_NUMBER=$(($(google-play get-latest-build-number --package-name "$PACKAGE_NAME" --tracks=alpha) + 1))
-              ./export_unity.sh android        
+              python3 export_unity.py android        
       artifacts:
           - android/*.aab
       publishing:
           google_play:
             # See the following link for information regarding publishing to Google Play - https://docs.codemagic.io/publishing-yaml/distribution/#google-play
             credentials: $GCLOUD_SERVICE_ACCOUNT_CREDENTIALS
-            track: alpha   # Any default or custom track that is not in ‘draft’ status    
+            track: alpha   # Any default or custom track that is not in ‘draft’ status
```

Comment:
Just saying that it's now also possible to publish as draft

Reply:
Removed comment about 'draft' status.